assets/src/edit-story/components/panels/design/borderRadius/borderRadius.js

```diff
@@ -143,51 +175,55 @@ function BorderRadiusPanel({ selectedElements, pushUpdateForObject }) {
     : __('Top left corner radius', 'web-stories');
   return (
     <SimplePanel name="borderRadius" title={__('Corner radius', 'web-stories')}>
-      <BorderRow>
-        <BoxedNumeric
-          value={borderRadius.topLeft}
-          aria-label={firstInputLabel}
-          onChange={(value) => handleChange('topLeft', value)}
-        />
-        {!lockRadius && (
-          <>
-            <Icon>
-              <Union />
-            </Icon>
-            <BoxedNumeric
-              value={borderRadius.topRight}
-              aria-label={__('Top right corner radius', 'web-stories')}
-              onChange={(value) => handleChange('topRight', value)}
+      <FlexContainer>
+        <InputContainer>
+          <Row>
+            <BoxedNumericInput
+              value={borderRadius.topLeft}
+              aria-label={firstInputLabel}
+              onChange={(_, value) => handleChange('topLeft', value)}
+            />
+            {!lockRadius && (
+              <>
+                <Icon>
+                  <Union />
```

Comment:
Don't have this icon in the design system. Should we move it from the editor to the design system?

Reply:
Let's make a debt ticket for that

Reply:
Added here: https://github.com/google/web-stories-wp/issues/6623